### PR TITLE
Documentation for MountPropagation beta

### PR DESCRIPTION
**Pull request by jsafrane**
https://github.com/kubernetes/features/issues/432

cc: @msau42

### CODE REVIEW

File: docs/reference/feature-gates.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ different Kubernetes components.
 | `LocalStorageCapacityIsolation` | `false` | Alpha | 1.7 | 1.9 |
 | `LocalStorageCapacityIsolation` | `true` | Beta| 1.10 | |
 | `MountContainers` | `false` | Alpha | 1.9 | |
-| `MountPropagation` | `false` | Alpha | 1.8 | |
```
Comment (tengqm) [inline docs/reference/feature-gates.md]: We keep tracking the status of each feature gates so you may want to revise this line AND add the following line:

```
| `MountPropagation` | `false` | Alpha | 1.8 | 1.9  |
```

Comment (jsafrane) [reply docs/reference/feature-gates.md]: Fixed

File: docs/reference/feature-gates.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ different Kubernetes components.
 | `LocalStorageCapacityIsolation` | `true` | Beta| 1.10 | |
 | `MountContainers` | `false` | Alpha | 1.9 | |
 | `MountPropagation` | `false` | Alpha | 1.8 | |
```
Comment (tengqm) [inline docs/reference/feature-gates.md]: nit ... line 63 should read

```
 | `MountPropagation` | `false` | Alpha | 1.8 | 1.9 |
```

This clearly indicates that the Alpha stage ends at 1.9 version.

**Comment by Bradamant3** [comment]
fixed merge conflict and responded to comment
/lgtm

**Pull request merged**